app/assets/stylesheets/components/_game-card.scss

```diff
@@ -0,0 +1,98 @@
+.game-card{
+  @media (max-width: 768px) {
+    display: none,
+  }
+
+    margin : auto;
+    margin-bottom: 3rem;
```

Comment:
Maybe we should use the [8 point rule](https://builttoadapt.io/intro-to-the-8-point-grid-system-d2573cde8632) for consistency